tests/injection/outjection.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+# SPDX-License-Identifier: MPL-2.0
+
+from typegraph import typegraph, Policy, t, Graph
+from typegraph.runtimes import RandomRuntime, DenoRuntime
+
+
+@typegraph()
+def outjection(g: Graph):
+    deno = DenoRuntime()
+    random = RandomRuntime(seed=1)
+
+    g.expose(
+        Policy.public(),
+        randomUser=deno.identity(t.struct()).extend(
+            {
+                "id": t.uuid().from_random(),
+                "age": t.integer().set(19),
+                "email": t.email().from_context("user_email"),
+                "password": t.string().from_secret(
+                    "user_password"
+                ),  ## should we allow this?
+                "createdAt": t.datetime().inject("now"),
```

Comment:
Does metagen handle this well btw? I.e. do we need to generate separate types without the injeted fields for the FDK/custom function implementers? I don't think it does, should I add it to the meagen papercuts tickeT?

Reply:
For injection on output types ("outjections"), it think it is not needed. But, it is indeed necessary for injections..

Reply:
But wouldn't the type system complain if you don't provide these outjected values in your custom function responses?

Reply:
Ah, you are right... It is needed for the FDK...